### ISSUE TITLE
Add delayed ossification of the clavicle

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10532]
**Preferred term label:**
Delayed ossification of the clavicle

**Synonyms**
-

**Definition (free text, please give PubMed ID)**
Delayed maturation and calcification of the clavicle.

**Parent term (use hpo.jax.org/app)**
Abnormal enchondral ossification HP:0003336

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
For example, cleidocranial dysplasia (omim199600) PMID12424591_1

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000874
- label: Delayed ossification of the clavicle
- parents: [Abnormal clavicle morphology (HP:0000889)](https://hpo.jax.org/app/browse/term/HP:0000889)
- pmid: 
- def: Delayed maturation and calcification of the clavicle.
- comment: 
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10532
- synonyms:
